src/System.Net.Security/tests/FunctionalTests/SslStreamNetworkStreamTest.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Net.Sockets;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+
+using Xunit;
+
+namespace System.Net.Security.Tests
+{
+    public class SslStreamNetworkStreamTest
+    {
+        [Fact]
+        public async void SslStream_SendReceiveOverNetworkStream_Ok()
+        {
+            X509Certificate2 serverCertificate = TestConfiguration.GetServerCertificate();
+            TcpListener listener = new TcpListener(IPAddress.Any, 4567);
```

Comment:
We should change the port from a fixed '4567' to a dynamic port, i.e. '0'.  Otherwise, if a single CI machine is running multiple instances of PR validation, this port might be busy running another test.